libraries/AC_PID/AC_PID.cpp

```diff
@@ -275,6 +277,8 @@ float AC_PID::update_all(float target, float measurement, float dt, bool limit,
     _pid_info.D = D_out;
     _pid_info.FF = _target * _kff;
     _pid_info.DFF = _target_derivative * _kdff;
+    _pid_info.dt = dt;
+    _pid_info.max_dt = MAX(dt, _pid_info.dt);
```

Comment:
This will presumably be the same for roll pitch and yaw? That is 4 redundant floats vs a time stamp and log header. New log message would use less log space especially if we ever wanted to expand logging of such things eg min dt.

Reply:
Yes agree, just WIP - will remove it when I have the bugs sorted

Reply:
Removed